zcash_client_backend/Cargo.toml

```diff
@@ -23,11 +23,15 @@ jubjub = "0.5.1"
 nom = "5.1.2"
 protobuf = "2.20"
 rand_core = "0.5.1"
+ripemd160 = { version = "0.9.1", optional = true }
+secp256k1 = { version = "0.19", optional = true }
+sha2 = "0.9"
 subtle = "2.2.3"
 time = "0.2"
 zcash_primitives = { version = "0.4", path = "../zcash_primitives" }
 proptest = { version = "0.10.1", optional = true }
 percent-encoding = "2.1.0"
+hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "72f1f7a56c114eed484cefd6d402b7ef28158712", optional = true }
```

Comment:
Perhaps we can/should remove your fork here since the upstream accepted your changes?